clang/include/clang/Sema/Overload.h

```diff
@@ -407,6 +407,18 @@ class Sema;
              Third == ICK_Identity;
     }
 
+    bool isPerfect(const ASTContext &C) const {
```

Comment:
I would LOVE a comment here, the implementation isn't particularly clear what is going on here.

Reply:
Still would like comments on these functions that say what 'perfect' means here.